lpc176x5x/lpc176x5x.svd

```diff
@@ -15662,12 +15661,12 @@ output edges.</description>
 					<enumeratedValues>
 					<name>ENUM</name>
 						<enumeratedValue>
-							<name>ERROR_OCCURRED_DURIN</name>
```

Comment:
:smile: This looks like an error happened while generating the svd file :wink:

Reply:
No, look at the description:

> Error occurred during transmitting.

Reply:
Yes, the joke was that there was *also* an error during generation because the value is truncated. :wink:

Reply:
Ah :smile: